wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -410,20 +413,42 @@ body.page .entry-header {
 	}
 }
 
+/* Override base styles */
+.navigation-top .wrap {
+	box-sizing: content-box;
```

Comment:
This is the default, see https://github.com/WordPress/wp20.wordpress.net/blob/trunk/wp-content/themes/twentyseventeen-wp20/style.css#L234